docs/api-references/user.md

```diff
@@ -0,0 +1,127 @@
+## User API
+
+### `GET` User
+```
+/api/user/
```

Comment:
Resource `user` should be `users`. You can check on: https://laravel.com/docs/5.4/controllers#resource-controllers